stubs/pywin32/win32/win32print.pyi

```diff
@@ -13,7 +15,7 @@ def GetDefaultPrinter() -> str: ...
 def GetDefaultPrinterW() -> str: ...
 def SetDefaultPrinter(printer: str): ...
 def SetDefaultPrinterW(Printer: str): ...
-def StartDocPrinter(hprinter: _win32typing.PyPrinterHANDLE, _tuple, level: int = ...): ...
+def StartDocPrinter(__hprinter: _win32typing.PyPrinterHANDLE | int, __level: Literal[1], __tuple: Sequence[str]): ...
```

Comment:
```suggestion
def StartDocPrinter(__hprinter: _win32typing.PyPrinterHANDLE | int, __level: Literal[1], __tuple: tuple[str, str, str]): ...
```
It's documented as a tuple, though from looking at the CPython code you are probably right that other sequences would work at runtime.

Reply:
I wouldn't rely too heavily on the documentation pages https://github.com/mhammond/pywin32#docs
> The docs are a long and sad story, but there's now an online version of the helpfile that ships with the installers. Lots of that is very old, but some is auto-generated and current. Would love help untangling the docs!

That said, the validation error message for the 3rd param mentions a "sequence" and I did validate that non-list non-tuple sequences work (like a string `"123"`)

Reply:
I think it's better to use `tuple[str, str, str]` here. The C argument parsing used accepts other objects recognized as sequences in C (which isn't the same concept as `Sequence` in Python), but tuples make the most sense for heterogeneous, fixed-length sequences. It's similar to how we type parameters as `bool` even if the C code will accept anything that is convertible to a bool.